src/pages/payment-methods/_mobile-expanded-list.js

```diff
@@ -236,8 +242,11 @@ const MobileExpandedList = ({ is_crypto, is_fiat_onramp, is_dp2p, locale, paymen
                     {/* reference row */}
                     <StyledRow jc="space-between" ai="center">
                         <StyledItemDiv>
-                            <Header type="subtitle-2">{localize('Reference')}</Header>
+                            <Header type="subtitle-2">
```

Comment:
```suggestion
                            <Header as="p" type="subtitle-2">
```

Reply:
Please include the `as` prop for the Header component if you notice any missing, otherwise they will default to a `h1` element when in this case it shouldn't be.